packages/babel-helpers/src/helpers/applyDecs2311.ts

```diff
@@ -197,6 +197,11 @@ export default /* @no-mangle */ function applyDecs2311(
   instanceBrand: Function,
   parentClass: any,
 ) {
+  var symbolMetadata = Symbol.metadata || Symbol.for("Symbol.metadata");
```

Comment:
We might at some point consider passing this as a parameter to `applyDecs2023` (with the `|| Symbol.for("Symbol.metadata")` fallback inside the helper), so that it can be polyfilled when using core-js-pure. Similarly to how we pass `Promise` when compiling async functions.

Reply:
![image](https://github.com/babel/babel/assets/30521560/03afbf28-59bf-43fa-9022-da5bad52aa52)
It seems to have been polyfilled now?